uftrace.c

```diff
@@ -860,6 +861,8 @@ static void parse_opt_file(int *argc, char ***argv, char *filename, struct opts
 	int orig_idx = 0;
 
 	if (stat(filename, &stbuf) < 0) {
+		if (keep_going)
+			return false;
```

Comment:
I think we can do the check in the call-site and get rid of this change.

Reply:
Okay. I will.